pystac/media_type.py

```diff
@@ -5,6 +5,7 @@ class MediaType(StringEnum):
     """A list of common media types that can be used in STAC Asset and Link metadata."""
 
     COG = "image/tiff; application=geotiff; profile=cloud-optimized"
+    FGB = "application/vnd.flatgeobuf"
```

Comment:
> https://github.com/stac-utils/pystac/pull/938#issuecomment-1373723750

In _my_ opinion, flatgeobuf is new enough that a short acronym is more confusing than helpful, so I agree that the full name is better.

```suggestion
    FLATGEOBUF = "application/vnd.flatgeobuf"
```

Plus, "FGB" doesn't roll off the tongue like "COG" does.

Can you also add a comment here linking back to the original flatgeobuf discussion where we settled on the `vnd.` value?

Reply:
Done.